features/cli/add_credential.feature

```diff
@@ -0,0 +1,65 @@
+Feature: Storing AWS access credentials
+
+  In order to manage AWS access credentials
+  As an AWS IAM user
+  I want to store an AWS access credential
+
+  Scenario: Adding a long-term access credential without an MFA device
```

Comment:
May also need to note that `.aws-session` does not exist, to differentiate from the "Adding a long-term access credential with an unknown MFA device" scenario below.